### DISCUSSION TITLE
Install and use

### DISCUSSION THREAD

**Discussion by zhouyuansheng0618** [FerretDB/FerretDB#277] [General]
Can you make a tutorial on building from scratch

**Comment by AlekSi** [comment]
@zhouyuansheng0618 Did you check https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md ?

**Comment by cwbeck** [comment]
I just want to be able to install this like a package. That is our use case at least...

How can we checkout, build and run the proxy (as you would in prod?). Don't mind the steps, but I don't want to consume this via docker.

**Comment by AlekSi** [comment]
Currently, https://github.com/FerretDB/FerretDB/blob/main/Taskfile.yml is the single source for our build steps.

Do you want to install via a .rpm or .deb package? Or via something else?

**Reply by cwbeck** [reply]
@AlekSi - ideally via a .deb package. It would make everything much easier :)

**Reply by AlekSi** [reply]
Created #734 and #735 to track the implementation. Currently, they are not planned, but we would gladly work with someone from the community who would be interested in implementing them.

**Comment by AlekSi** [comment]
We provide .deb and .rpm packages now; see assets at https://github.com/FerretDB/FerretDB/releases/tag/v0.5.1

**Reply by gaoyan0204** [reply]
How to configure ferretdb for trial use after NPM installation？

**Reply by AlekSi** [reply]
Did you try our [quickstart guide](https://github.com/FerretDB/FerretDB#quickstart)?

**Reply by gaoyan0204** [reply]
I want to connect feretdb with mongodb client. What is the default user name and password？

**Reply by AlekSi** [reply]
There is no authentication support yet, so both could be empty
